src/index.html

```diff
@@ -1,23 +1,23 @@
 <!DOCTYPE html>
-<html>
+<html lang='en'>
   <head>
     <meta charset="UTF-8">
     <title>Activity Tracker</title>
     <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans:400,800&display=swap" rel="stylesheet" />
   </head>
   <body>
-    <section class="main">
-      <section class="sidebar-container" id="user-sidebar">
-        <h1 id="headerText">Activity Tracker</h1>
-        <img src="./images/arnie.jpg" class="sidebar-header-userImage"></img>
-        <a href="">Profile</a>
+    <section class="main" tabindex="0">
+      <section class="sidebar-container" id="user-sidebar" tabindex="0">
+        <h1 id="headerText" tabindex="0">Activity Tracker</h1>
```

Comment:
I'd love a little info on what the affect the value of `tabindex` has.

Reply:
`tabindex=0` places the element in the logical navigation flow according to how a screenreader would read it, but changing the value to anything other than 0 will essentially force a specific order. According to everything I looked up about it, that is usually bad practice and not preferred.